File: CS/Step 1 - creating model/Helpers/PersonsCreator.cs
using DX_MVVM.Models;

namespace DX_MVVM.Helpers {
    public static class PersonsCreator {
        public static readonly Persons Persons;

        static PersonsCreator() {
            Persons = new Persons();
            foreach(Person person in PersonCreator.Person)
                Persons.Add(person);
        }
    }
}

File: VB/Step 4 - building views/Helpers/PersonsCreator.vb
Imports DX_MVVM.Models

Namespace DX_MVVM.Helpers
    Public NotInheritable Class PersonsCreator

        Private Sub New()
        End Sub
        Public Shared ReadOnly Persons As Persons

        Shared Sub New()
            Persons = New Persons()
            For Each person As Person In PersonCreator.Person
                Persons.Add(person)
            Next person
        End Sub
    End Class
End Namespace
